src/overlays/actors/ovl_Bg_Po_Event/z_bg_po_event.c

```diff
@@ -102,7 +102,7 @@ void BgPoEvent_InitPaintings(BgPoEvent* this, PlayState* play) {
     static s16 paintingPosX[] = { -1302, -866, 1421, 985 };
     static s16 paintingPosY[] = { 1107, 1091 };
     static s16 paintingPosZ[] = { -3384, -3252 };
-    ColliderTrisElementInit* item;
+    ColliderTrisElementInit* elementInit;
     Vec3f* vtxVec;
```

Comment:
sometimes element is spelled out and sometimes not (like below)
I dont have a preference but is there a reason

Reply:
no particular reason, it may have been an intuitive choice because the shortened one is referenced more but idk

Reply:
i feel like there should probably be a standard but if you dont want to do that in this pr thats fine

Reply:
changed the shortened "elem" ones to full "element"